### PR TITLE
Calibration: Implementation of FuelClient related functions

### PR DESCRIPTION
**Pull request by changsh726**

### CODE REVIEW

File: modules/dreamview/backend/fuel_monitor/preprocess_monitor.cc [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        this->OnProgress(progress);
+      });
+}
+
+void PreprocessMonitor::LoadConfiguration() {
+  if (!task_name_.empty()) {
+    const std::string& vehicle_dir =
+        VehicleManager::Instance()->GetVehicleDataPath();
+    std::string config_path = absl::StrCat(
+        vehicle_dir, "dreamview_conf/", task_name_, "_precrocess_table.pb.txt");
```
Comment (lj0919) [inline modules/dreamview/backend/fuel_monitor/preprocess_monitor.cc:72]: preprocess

Comment (changsh726) [reply modules/dreamview/backend/fuel_monitor/preprocess_monitor.cc]: thx

File: modules/dreamview/conf/hmi_modes/vehicle_calibration.pb.txt [hunk 2]
```diff
@@ -55,8 +71,21 @@ monitored_components {
...
+  }
+}
+other_components {
+  key: "Preprocess"
   value: {
-    # Special LocalizationMonitor.
+    command_keywords: "bash"
+    command_keywords: "preprocess.sh"
   }
 }
```
Comment (macDure) [inline modules/dreamview/conf/hmi_modes/vehicle_calibration.pb.txt:91]: confirm the vehicle calibration config

File: modules/tools/vehicle_calibration/preprocess.py [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+            self.progress.status = preprocess_table_pb2.Status.SUCCESS
+        else:
+            self.progress.status = preprocess_table_pb2.Status.FAIL
+        self.progress.log_string = log_str
+        self.writer.write(self.progress)
+        time.sleep(0.5)
+
+
+if __name__ == "__main__":
+    app.run(main)
```
Comment (macDure) [inline modules/tools/vehicle_calibration/preprocess.py:178]: reviewed

File: modules/tools/vehicle_calibration/sanity_check.py [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+    elif channel_flag:
+        err_msg = (
+            'Messages are missing in records channels apollo/chassis or '
+            f'apollo/localization/pose, records are {channel_result}')
+    else:
+        info_msg = f'{input_folder} Passed sanity check.'
+        logging.info(info_msg)
+        return True, info_msg
+    logging.error(err_msg)
+    return False, err_msg
```
Comment (macDure) [inline modules/tools/vehicle_calibration/sanity_check.py:184]: the same with fuel

**Review by macDure — DISMISSED** [review]

**Review by SeasoulChris — DISMISSED** [review]
looks good to me

**Comment by changsh726** [comment]
resolved confilcts

**Review by lfcarol — APPROVED** [review]

**Pull request merged**